### DISCUSSION TITLE
any update on H200 integration ?

### DISCUSSION THREAD

**Discussion by alx-xlx** [petretiandrea/home-assistant-tapo-p100#819] [Q&A]
I have been waiting for the h200 hub integration still cannot see if its integrated or not.

**Comment by Lumpsack** [comment]
I'm hoping so too, would be great to get that working - guessing I'd get access to the media then.

**Comment by serkaneskizara** [comment]
I'm also looking forward for h200

**Comment by luketechnology-hue** [comment]
Just wanted to say.... im also looking forward to the H200 working in HA... to that end i have a H200 and plenty of time... is there some diagnostics or testing etc that i can do on my H200 that will help things along/get you the data you need?... please let me know as it would be great to get this working in HA... thanks
